.pre-commit-hooks.yaml

```diff
@@ -28,3 +28,12 @@
   pass_filenames: false
   additional_dependencies: []
   minimum_pre_commit_version: "2.9.2"
+- id: uv-sync
+  name: uv-sync
+  description: "Automatically run 'uv sync' post merge or checkout"
+  entry: uv sync
+  args: ["--locked"]
```

Comment:
I'm wondering if this should be `--frozen`.

Currently if you checkout a branch with an outdated lockfile, it doesnt sync the env but errors.
With `--frozen`, it would sync the env but hide that the lockfile is not up to date (which is maybe not that good)